jcl/src/java.base/share/classes/java/lang/Object.java

```diff
@@ -294,6 +294,21 @@ public final void wait(long time, int frac) throws InterruptedException {
 			Blocker.end(blocking);
 		}
 	}
+/*[ELSE] JAVA_SPEC_VERSION < 24 */
+	if ((time < 0) || (frac < 0)) {
+		throw new IllegalArgumentException("timeout value is negative");
+	}
+	if (!Thread.currentThread().isVirtual()) {
+		waitImpl(time, frac);
+	} else {
+		try {
+			waitImpl(time, frac);
+		} catch (InterruptedException e) {
+			// Clear virtual thread's interrupt status
+			Thread.currentThread().getAndClearInterrupt();
+			throw e;
+		}
+	}
 /*[ENDIF] JAVA_SPEC_VERSION < 19 */
```

Comment:
The version comment here looks out of date.

Reply:
FYI @fengxue-IS

Reply:
This is the closing for line 277, not sure what should I put here as there are 4 different JDK versions

Reply:
The existing code matches the first `/*[IF` which is `JAVA_SPEC_VERSION < 19` in this case.
It looks fine to me.